extensions/wikia/Wall/js/WallMessageForm.js

```diff
@@ -1,5 +1,5 @@
 /* global Wall:true, UserLoginAjaxForm, UserLoginModal, Observable */
-(function (window, $) {
+(function (window, $, mw) {
```

Comment:
Without also adding `mediaWiki` to the IIFE, `mw` will be undefined when it's used to call `mw.hook` in this file.